doc/licenses/string_decoder-1.1.1/README

```diff
@@ -0,0 +1,8 @@
+string_decoder (https://github.com/nodejs/string_decoder)
+---------------------------------------------
+
+    Version: 1.1.1
+    From: 'Node.js' (https://github.com/nodejs)
```

Comment:
According to the license, this should be "Node.js contributors, Joyent, Inc., and other Node contributors".